debian/nfd.service

```diff
@@ -32,6 +32,8 @@ After=nfd-keygen.service
 [Service]
 EnvironmentFile=-/etc/default/nfd
 Environment=HOME=/var/lib/ndn
+Environment=LANG=C
+Environment=LC_ALL=
```

Comment:
You can just set `LC_ALL=C` and override everything. In fact, this would not cover the case where the locale setup has `LC_MESSAGES=missing-locale` since that supersedes `LANG`.